pss/pss.go

```diff
@@ -739,7 +737,7 @@ func sendMsg(p *Pss, sp *network.Peer, msg *message.Message) bool {
 //// forwarding fails, the node should try to forward it to the next best peer, until the message is
 //// successfully forwarded to at least one peer.
 func (p *Pss) forward(msg *message.Message) error {
-	metrics.GetOrRegisterCounter("pss.forward", nil).Inc(1)
+	defer metrics.GetOrRegisterResettingTimer("pss.forward.time", nil).UpdateSince(time.Now())
```

Comment:
No need to add a `time` suffix to metrics names. This will be emitted as `pss.forward.span` and `pss.forward.count` by the reporter - the time part is implicit when you register a resetting timer.